pkg/fileutil/files.go

```diff
@@ -173,3 +173,45 @@ func GetFilesWithSpecificExtension(folderPath, ext string) []string {
 
 	return files
 }
+
+func AddLineToFile(filePath, lineText, commentText string) error {
+	f, err := os.OpenFile(filePath, os.O_APPEND|os.O_CREATE|os.O_WRONLY, os.ModePerm) //nolint:gomnd
+	if err != nil {
+		return err
+	}
+	content, err := os.ReadFile(filePath)
+	if err != nil {
+		return err
+	}
+	if !strings.Contains(string(content), lineText) {
+		_, err = f.WriteString("\n" + lineText + " " + commentText)
```

Comment:
```suggestion
		_, err = f.WriteString(lineText + " " + commentText + "\n")
```
should it end with an end of the line?

Reply:
not sure but it definitely needs a `"\n" +` up front if the users file doesn't end their file with a new line this won't work

Reply:
in this particular use case didn't think we needed to add a new line at the end since we are going to delete it later